src/Illuminate/Database/Eloquent/Relations/Relation.php

```diff
@@ -294,9 +293,21 @@ public function getQuery()
     /**
      * Get the base query builder driving the Eloquent builder.
      *
+     * @deprecated Use toBase() instead
+     *
      * @return \Illuminate\Database\Query\Builder
      */
     public function getBaseQuery()
+    {
+        return $this->toBase();
+    }
```

Comment:
Tbvh this method can be removed and deprecated in 8.x instead. But up to @taylorotwell.